client/src/components/Festivals.js

```diff
@@ -25,22 +32,44 @@ const FestivalIntroduction = styled.div`
   font-size: 12px;
   font-family: 'Quicksand Light';
   line-height: 1.3;
-  margin-bottom: 30px;
+  margin-bottom: 20px;
   text-transform: uppercase;
 `;
 
+const LineUpHeader = styled.div`
+  font-size: 20px;
+  margin-bottom: 20px;
+`;
+
 const LineUp = styled.div`
   display: flex;
   flex-wrap: wrap;
-  font-size: 20px;
-  margin-bottom: 20px;
-  font-size: 12px;
-  font-family: 'Quicksand Book';
-  text-transform: uppercase;
+  background-image: linear-gradient(
+    145deg,
+    rgba(255, 247, 0, 0.5),
+    rgba(255, 145, 0, 0.5),
+    rgba(255, 0, 132, 0.5),
+    rgba(255, 0, 242, 0.5),
+    rgba(187, 0, 255, 0.5),
+    rgba(85, 0, 255, 0.5),
+    rgba(0, 106, 255, 0.5),
+    rgba(0, 204, 255, 0.5),
+    rgba(0, 255, 106, 0.5),
+    rgba(89, 255, 0, 0.5),
+    rgba(255, 247, 0, 0.5)
+  );
+`;
+const Cell = styled.div`
+  padding: 2px;
+  overflow: hidden;
+`;
+const Fill = styled.div`
+  flex-grow: 1;
+  background: #f9f9f9;
 `;
 
 async function fetchFestivals() {
-  const response = await fetch('http://localhost:4000/api/festivals');
+  const response = await fetch('/festivals');
```

Comment:
Make sure the API runs in `/api`. You might have to adjust your server (API) too. 

```suggestion
  const response = await fetch('/api/festivals');
```

Reply:
Whenever I set it to '/api/festivals' I get a 404.